<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt4ProjectManager::Internal::GettingStartedWelcomePageWidget</class>
 <widget class="QWidget" name="Qt4ProjectManager::Internal::GettingStartedWelcomePageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="2" column="0">
    <widget class="QFrame" name="contentframe">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="3">
       <widget class="QFrame" name="tutorialsFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="verticalSpacing">
          <number>12</number>
         </property>
         <item row="0" column="0">
          <widget class="Utils::WelcomeModeLabel" name="tutorialsTitleLabel">
           <property name="text">
            <string>Tutorials</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="Utils::WelcomeModeTreeWidget" name="tutorialTreeWidget" native="true">
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="demosExamplesFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="Utils::WelcomeModeLabel" name="demoTitleLabel">
           <property name="text">
            <string>Explore Qt C++ Examples</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cppExamplesButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Examples not installed...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="demosExamplesFrame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="Utils::WelcomeModeLabel" name="demoTitleLabel_2">
           <property name="text">
            <string>Explore Qt Quick Examples</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qmlExamplesButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Examples not installed...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFrame" name="didyouKnowFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_11">
         <property name="verticalSpacing">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_10">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0" colspan="2">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>13</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="nextTipBtn">
             <property name="styleSheet">
              <string notr="true">QToolButton{
	border-left:solid 0 px;
	height:16px;
	width:12px;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../welcome/welcome.qrc">
               <normaloff>:/welcome/images/arrow-right.png</normaloff>:/welcome/images/arrow-right.png</iconset>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QToolButton" name="prevTipBtn">
             <property name="styleSheet">
              <string notr="true">QToolButton{
	border-right:solid 0 px;
	height:16px;
	width:12px;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../welcome/welcome.qrc">
               <normaloff>:/welcome/images/arrow-left.png</normaloff>:/welcome/images/arrow-left.png</iconset>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>13</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QTextBrowser" name="didYouKnowTextBrowser">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="Utils::WelcomeModeLabel" name="didYouKnowTitleLabel">
             <property name="text">
              <string>Did You Know?</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="openProjectButton">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>Open Project...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createNewProjectButton">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>36</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="text">
           <string>Create Project...</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::WelcomeModeTreeWidget</class>
   <extends>QWidget</extends>
   <header location="global">utils/welcomemodetreewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Utils::WelcomeModeLabel</class>
   <extends>QLabel</extends>
   <header location="global">utils/welcomemodetreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../welcome/welcome.qrc"/>
 </resources>
 <connections/>
</ui>
